{{cookiecutter.project_dirname}}/scripts/deploy/terraform.sh

```diff
@@ -1,7 +1,9 @@
-#!/bin/sh -e
+#!/usr/bin/env sh
+
+set -e
 
 if [ "${DEBUG_OUTPUT}" = "true" ]; then
-    set -x
+    set -ex
```

Comment:
```suggestion
    set -x
```
`set` options work independently.